.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Since this is the default for directory, we shouldn't have to specify it. Which version of bower are you using?